packages/contractkit/src/wrappers/ReleaseGold.ts

```diff
@@ -149,13 +149,24 @@ export class ReleaseGoldWrapper extends BaseWrapper<ReleaseGold> {
    * @return A RevocationInfo struct.
    */
   async getRevocationInfo(): Promise<RevocationInfo> {
-    const revocationInfo = await this.contract.methods.revocationInfo().call()
-
-    return {
-      revocable: revocationInfo.revocable,
-      canExpire: revocationInfo.canExpire,
-      releasedBalanceAtRevoke: valueToBigNumber(revocationInfo.releasedBalanceAtRevoke),
-      revokeTime: valueToInt(revocationInfo.revokeTime),
+    try {
+      const revocationInfo = await this.contract.methods.revocationInfo().call()
+      return {
+        revocable: revocationInfo.revocable,
+        canExpire: revocationInfo.canExpire,
+        releasedBalanceAtRevoke: valueToBigNumber(revocationInfo.releasedBalanceAtRevoke),
+        revokeTime: valueToInt(revocationInfo.revokeTime),
+      }
+    } catch (_) {
+      // This error is caused by a mismatch between the deployed contract and the locally compiled version.
```

Comment:
Can we be sure to track this hack in an issue so it can be removed?

Reply:
Yup will do